app/scripts/utils/polygon-url.ts

```diff
@@ -1,27 +1,29 @@
-import { Feature, MultiPolygon } from 'geojson';
+import { FeatureCollection, Polygon } from 'geojson';
 import gjv from 'geojson-validation';
 import { decode, encode } from 'google-polyline';
 
 /**
- * Decodes a multi polygon string converting it into a MultiPolygon feature.
+ * Decodes a multi polygon string converting it into a FeatureCollection of
+ * Polygons.
  *
- * lon,lat|lon,lat||lon,lat|lon,lat
- * || separates polygons
- * | separates points
+ * lon,lat|lon,lat||lon,lat|lon,lat || separates polygons | separates points
```

Comment:
This comment is not true anymore since we used `google-polyline`

Reply:
indeed 💯